gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Jan 26 14:32:06 IST 2019
+#Fri May 17 15:10:56 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
Can be increased to 5.4.1

Reply:
Android studio automatically downloaded it to 5.1.1 version after it was upgraded from 3.3.2 to 3.4.1. So, I don't think it will be much helpful. Also, the build is successfully taking place in my system.

Reply:
Upgrade to 5.4.1 please. It's the latest version

Reply:
okk sir.